macros/EmbedInteractiveExample.ejs

```diff
@@ -28,4 +29,4 @@ if ($1) {
 }
 
 %>
-<iframe class="<%= iClass %>" width="100%" height="250" frameborder="0" src="<%= url %>" title="MDN Web Docs Interactive Example"></iframe>
+<iframe class="<%= iClass %>" width="100%" height="250" frameborder="0" src="<%= url %>" title="Interactive Example"></iframe>
```

Comment:
`title="MDN Web Docs Interactive Example"` should be changed to `title="<%= s_title %>"`